oxidation/libparsec/crates/protocol/schema/authenticated_cmds/events_listen.json

```diff
@@ -134,9 +134,14 @@
                             },
                             {
                                 "name": "role",
-                                "type": "RealmRole"
+                                "type": "Option<RealmRole>"
                             }
                         ]
+                    },
+                    {
+                        "name": "Pki_enrollment.updated",
```

Comment:
name is wrong, should be `PkiEnrollmentUpdated` I guess

Reply:
done